src/components/PeopleFilters.tsx

```diff
@@ -1,19 +1,57 @@
-export const PeopleFilters = () => {
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+
+interface Props {
+  sex: string | null,
+}
+
+export const PeopleFilters: React.FC<Props> = ({
+  sex,
+}) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [query, setQuery] = useState<string | null>(null);
+  const centuries = searchParams.getAll('centuries') || [];
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <Link
+          className={classNames({ 'is-active': !sex })}
+          to={{ search: getSearchWith(searchParams, { sex: null }) }}
+        >
+          All
+        </Link>
+        <Link
+          className={classNames({ 'is-active': sex === 'm' })}
+          to={{ search: getSearchWith(searchParams, { sex: 'm' }) }}
```

Comment:
It would be cool not to use magic values and put such strings into constants and reuse them:
```
const MALE = 'm';
const FEMALE = 'f';
```
This will increase the readability of the code and save us from possible errors.